ontology/rml-core.owl

```diff
@@ -0,0 +1,517 @@
+
+
+###  http://w3id.org/rml/IRI
+:IRI rdf:type owl:Class ;
+     rdfs:comment "Denotes an IRI, used with termpType."@en ;
+     rdfs:isDefinedBy <http://w3id.org/rml/core/> ;
+     rdfs:label "IRI" .
+
+
+###  http://w3id.org/rml/Join
+:Join rdf:type owl:Class ;
+      rdfs:subClassOf owl:Thing ,
+                      [ rdf:type owl:Restriction ;
+                        owl:onProperty :child ;
+                        owl:qualifiedCardinality "1"^^xsd:nonNegativeInteger ;
+                        owl:onDataRange xsd:string
+                      ] ,
+                      [ rdf:type owl:Restriction ;
+                        owl:onProperty :parent ;
+                        owl:qualifiedCardinality "1"^^xsd:nonNegativeInteger ;
+                        owl:onDataRange xsd:string
+                      ] ;
+      rdfs:comment "Represents a join condition."@en ;
+      rdfs:isDefinedBy <http://w3id.org/rml/core/> ;
+      rdfs:label "Join" .
+
+
+###  http://w3id.org/rml/LanguageMap
+:LanguageMap rdf:type owl:Class ;
+             rdfs:subClassOf :ExpressionMap ;
+             rdfs:comment "Represents a language map" ;
+             rdfs:isDefinedBy <http://w3id.org/rml/core/> ;
+             rdfs:label "Language Map" .
+
+
+###  http://w3id.org/rml/Literal
+:Literal rdf:type owl:Class ;
+         rdfs:subClassOf owl:Thing ;
+         rdfs:comment "Denotes a Literal, used with termType."@en ;
+         rdfs:isDefinedBy <http://w3id.org/rml/core/> ;
+         rdfs:label "Literal" .
+
+
+###  http://w3id.org/rml/LogicalSource
+:LogicalSource rdf:type owl:Class ;
+               rdfs:comment "Represents a logical source." ;
+               rdfs:isDefinedBy <http://w3id.org/rml/core/> ;
+               rdfs:label "Logical Source" .
+
+
+###  http://w3id.org/rml/LogicalTarget
+:LogicalTarget rdf:type owl:Class ;
+               rdfs:comment "Represents a Logical Target" ;
+               rdfs:isDefinedBy <http://w3id.org/rml/core/> ;
+               rdfs:label "Logical Target" .
+
+
+###  http://w3id.org/rml/ObjectMap
+:ObjectMap rdf:type owl:Class ;
+           rdfs:subClassOf :TermMap ,
+                           [ rdf:type owl:Restriction ;
+                             owl:onProperty :datatype ;
+                             owl:minQualifiedCardinality "0"^^xsd:nonNegativeInteger ;
+                             owl:onClass rdfs:Datatype
+                           ] ,
+                           [ rdf:type owl:Restriction ;
+                             owl:onProperty :object ;
+                             owl:minQualifiedCardinality "0"^^xsd:nonNegativeInteger ;
+                             owl:onClass rdfs:Resource
+                           ] ,
+                           [ rdf:type owl:Restriction ;
+                             owl:onProperty :object ;
+                             owl:maxQualifiedCardinality "1"^^xsd:nonNegativeInteger ;
+                             owl:onClass rdfs:Resource
+                           ] ,
```

Comment:
These belong on `:PredicateObjectMap`. And I don't see why we should restrict the cardinality there. So this should not be bound IMO.